tests/unittests/benchmark/task/profet/conftest.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+import pytest
+from orion.benchmark.task.profet.profet_task import MetaModelTrainingConfig, NAMES
+
+REAL_PROFET_DATA_DIR: Path = Path("profet_data")
+
+requires_profet_data = pytest.mark.skipif(
+    not REAL_PROFET_DATA_DIR.exists(),
+    reason="Need to have the real profet data downloaded to run this test.",
+)
+
+
+@pytest.fixture(scope="session")
+def profet_train_config():
+    """ Fixture that provides a configuration object for the Profet algorithm for testing. """
+    # TODO: Figure out a good set of values that makes the training of the meta-model really fast,
```

Comment:
Issue or worth fixing here? How long are the profet tests?

Reply:
Removing the todo, since the todo is solved: the tests are running fairly quickly now (I takes about 1 minute to run all the tests in `tests/unittests/benchmark`)

Reply:
Marking this as resolved.